### ISSUE TITLE
Revamp of code to use go string manipulation instead of system command line tools

### ISSUE THREAD

**Issue by abstractninjatur** [arunsathiya/set-output-janitor#2]
The code contains the logic to invoke system command line tools to find and replace the set-output line. Instead of this use go string manipluation.

**Comment by sharunkumar** [comment]
Just adding my 2 cents here: doing it in go will add more complexity like you would have to handle different new line characters, encodings, etc. Using `sed` I think, already does all of that for you

**Comment by arunsathiya** [comment]
There are definitely pros and cons to each approach! I am going to ponder on this for a few days and make changes, and most specifically, I want to refactor the current code to be concise.

**Comment by abstractninjatur** [comment]
> Just adding my 2 cents here: doing it in go will add more complexity like you would have to handle different new line characters, encodings, etc. Using `sed` I think, already does all of that for you

Yeah, True, IMO we can use  shell script instead of go to use the system commands like sed.
